chat-widget/src/components/livechatwidget/interfaces/ILiveChatWidgetProps.ts

```diff
@@ -54,6 +54,7 @@ export interface ILiveChatWidgetProps {
     preChatSurveyPaneProps?: IPreChatSurveyPaneProps;
     proactiveChatPaneProps?: IProactiveChatPaneStatefulProps;
     reconnectChatPaneProps?: IReconnectChatPaneStatefulProps;
+    errorPaneProps?: ILoadingPaneProps;
     startChatErrorPaneProps?: IStartChatErrorPaneProps;
```

Comment:
Why do we have `startChatErrorPaneProps` as well as `errorPaneProps`? Do we have other error pane types?

Reply:
From the props shown in description, doesn't it make more sense to put the controlProps of `startChatErrorPaneProps` inside `errorPaneProps`?

Reply:
Refactored, see other comment.